goon/browserassets/js/browserOutput.js

```diff
@@ -93,19 +91,6 @@ if (typeof String.prototype.trim !== 'function') {
 	};
 }
 
-//Polyfill for string.prototype.includes. Why the fuck. Just why the fuck.
```

Comment:
Why remove the includes polyfill?

Reply:
It is not needed.
It was added in #12655 
String.includes call is removed in both this PR and in your #12690

Reply:
it still was a nice thing to have regardless
~~babel on text panel when~~

Reply:
Let's not bring the entire js ecosystem into paradise just because it's "nice to have"

Reply:
I know, its just a joke. Anyways, i could see includes being used in other places (the whole admin filter bit could probably be rewritten using it)